scripts/generate-lint.go

```diff
@@ -58,6 +58,11 @@ func targetsFromChangedFiles(files []string) ([]string, error) {
 			if dir == scriptsDir {
 				continue
 			}
+
+			if strings.Contains(dir, "vendor") {
```

Comment:
shouldn't this be HasPrefix vendor instead?

Reply:
I wish it did. 

Alas, it's more fully qualified, so the strings are like: 

     github.com/evergreen-ci/evergreen/vendor/github.com/mongodb/grip 

rather than with the first bit chopped off.

Reply:
got it! thanks!